src/main/java/vbn/state/constraints/UnaryConstraint.java

```diff
@@ -16,11 +16,13 @@ public UnaryConstraint(@NonNull UnaryOperand op, @NonNull Symbol symbol) {
         this.assigned = null;
         this.symbol = symbol;
         this.op = op;
+        this.negated = false;
```

Comment:
```suggestion
```

Reply:
Fixed in [c20af33](https://github.com/CMPT-479/CMPT-479-Project/pull/32/commits/c20af330db3eb5857b775153c9499980e5da16a4) and [cefec5c](https://github.com/CMPT-479/CMPT-479-Project/pull/32/commits/cefec5c6ca5e3350d7c07a128c30b406a65d884a)